docs/source/userguide/user_interface_overview.rst

```diff
@@ -0,0 +1,49 @@
+An Introduction to the User Interface
+=====================================
+
+The user interface provides three principle views:
+
```

Comment:
The user interface is made of 3 panes, left, middle and right.

* Layout View: Parent Block in left with "layout" selected, child Block summary and connections in middle, selected child block details in right
* Attribute View: Block in left with Attribute selected, summary of Attribute value in the middle, more details on the right. Middle pane depends on type of Attribute, might be graph + table of historical values for scalars, or table of current value for tables